README

```diff
@@ -74,6 +74,10 @@ You will need to install required packages on the build system:
 * **numactl-devel**
 * **unzip**
 
+For using HMAT features in memkind:
```

Comment:
I suggest adding a short description when "HMAT" appears the first time. What it gives to memkind and happens if it is not present.